tle/cogs/codeforces.py

```diff
@@ -33,13 +34,28 @@ def __init__(self, bot):
         rc = cf_common.user_db.update_status(active_ids)
         await ctx.send(f'{rc} members active with handle')
 
-    async def _gitgud(self, ctx, handle, problem, delta):
+    async def _validate_gitgud_status(self, ctx, delta):
+        if delta % 100 != 0:
+            await ctx.send('Delta must be a multiple of 100.')
+            return False
+
+        if abs(delta) > _GITGUD_MAX_ABS_DELTA_VALUE:
+            await ctx.send(f'Delta must range from -{_GITGUD_MAX_ABS_DELTA_VALUE} to {_GITGUD_MAX_ABS_DELTA_VALUE}.')
```

Comment:
here as well. some of these functions were written very long time ago and they weren't updated to use that